resources/views/components/button/wishlist.blade.php

```diff
@@ -0,0 +1,22 @@
+@props(['productId' => $product->entity_id ?? null])
+
+<toggler v-slot="{ isOpen, toggle, close}" v-cloak>
```

Comment:
While you're here can you add a space on the right side of the slot definition? 😇